src/plasmapy/particles/atomic.py

```diff
@@ -1143,27 +1143,33 @@ def _is_electron(arg: Any) -> bool:
 @particle_input
 @validate_quantities(energies=u.MeV)
 def stopping_power(
-    incident_particle: Particle,
+    incident_particle: ParticleLike,
     material: str,
     energies: u.Quantity[u.MeV] | None = None,
-    component="total",
+    component: Literal["total", "electronic", "nuclear"] = "total",
 ) -> tuple[u.Quantity, u.Quantity]:
-    """Calculate stopping powers for a provided particle in a provided material.
+    """
+    Calculate stopping powers for a provided particle in a provided
+    material.
```

Comment:
🎨 To follow the convention to have docstrings justified to 72 characters.

🌠 I really wish we had an auto-formatter that would do this for us!